requirements.txt

```diff
@@ -1,3 +1,3 @@
 cairocffi >= 0.9.0
 cffi >= 1.1.0
-xcffib >= 0.10.1
```

Comment:
This is a bit naughty. I removed `xcffib` from here as it needs to be installed before `cairocffi`. I'm not sure this file would work anyway if anyone tried to install this file.

Reply:
I don't remember the exact message but it was to do with cairo not having xcb support.